packages/pwa/app/components/_app/index.jsx

```diff
@@ -228,30 +234,34 @@ const App = (props) => {
                             </Box>
 
                             {!isOnline && <OfflineBanner />}
-
-                            <SkipNavContent
-                                style={{
-                                    display: 'flex',
-                                    flexDirection: 'column',
-                                    flex: 1,
-                                    outline: 0
-                                }}
-                            >
-                                <Box
-                                    as="main"
-                                    id="app-main"
-                                    role="main"
-                                    display="flex"
-                                    flexDirection="column"
-                                    flex="1"
+                            <AddToCartModalContext.Provider value={addToCartModal}>
```

Comment:
What is the reason to have this modal context at the app level? 

From what I understand is that this modal will open when an item is added to cart? it should be short-lived and has nothing to do with other pages.